### PR TITLE
Changed db to sqlite3

### PR DESCRIPTION
**Pull request by Aman-15**

### CODE REVIEW

File: data/setup.sql [hunk 1]
```diff
@@ -8,51 +6,51 @@ drop table sessions;
 drop table users;
 
 create table users (
-  	id         serial primary key,
```
[the remark this answers is not in the document]
Comment (Aman-15) [reply data/setup.sql]: Problem with my editor, my bad!

File: data/user.go [hunk 2]
```diff
@@ -23,62 +23,103 @@ type Session struct {
...
 	}
-	return &sess, err
+	return &Session{id.Int64, uuid.String, userid.Int64, createdAt}, err
 }
 
 func GetUserByEmail(email string) (*User, error) {
-	user := User{}
-	err := Db.QueryRow("select id, uuid, name, email, password_hash, created_at, role from users where email = $1", email).
-		Scan(&user.Id, &user.Uuid, &user.Name, &user.Email, &user.Password_Hash, &user.CreatedAt, &user.Role)
+	var id sql.NullInt64
```
[the remark this answers is not in the document]
Comment (Aman-15) [reply data/user.go]: Taking input from sqlite is new. For example, for taking id (int64) as input from the select query, we need an sql.NULLInt64 type variable.

Although, we can use go-gorp for further abstraction. Let's hold that for later.

**Comment by Aman-15** [comment]
Not able to run tests, no logs are generated.

> .\github.com\suyashkumar2409\table-topics>make test
process_begin: CreateProcess(NULL, pwd, ...) failed.
process_begin: CreateProcess(NULL, pwd, ...) failed.
go test -v ./... -config=/config/test_config.json -log=/app.log
?       github.com/suyashkumar2409/table-topics [no test files]
?       github.com/suyashkumar2409/table-topics/config  [no test files]
2018/06/26 23:45:15 Failed to open config file open /config/test_config.json: The system cannot find the path specified.
FAIL    github.com/suyashkumar2409/table-topics/data    0.274s
?       github.com/suyashkumar2409/table-topics/route   [no test files]
make: *** [test] Error 1

**Comment by suyashkumar2409** [comment]
Looking at your log- it says process_begin: CreateProcess(NULL, pwd, ...) failed.
means it can't run pwd on a windows system
Look at the config flag - -config=/config/test_config.json
It doesn't have a full blown path starting from C:/Users/...etc
I guess we will have to create to check within makefile what the current OS is, and do different things from windows and MAC :(
Check this out - https://stackoverflow.com/questions/714100/os-detecting-makefile

do an if else as shown there, put the part that I have written in makefile under else so that it runs on my system.

**Pull request merged**